src/main/java/mate/academy/mybookshop/entity/BookEntity.java

```diff
@@ -41,4 +49,15 @@ public class BookEntity {
 
     @Column(nullable = false)
     private boolean isDeleted = false;
+
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    @JoinTable(
+            name = "books_categories",
+            joinColumns = @JoinColumn(
+                    name = "book_id", referencedColumnName = "id"),
+            inverseJoinColumns = @JoinColumn(
+                    name = "category_id", referencedColumnName = "id"))
```

Comment:
As I know, referencedColumnName has default value `id`

Reply:
yeah, and now  i know that too, but still good practice to show that.